reference/providers/kubernetes-v2/index.md

```diff
@@ -104,19 +104,85 @@ Serveral annotations are built into each manifest and cannot be used otherwise.
-Kubernetes supports. Also the Kubernetes extension mechanisms&mdash;called
-[Custom Resource Definitions
+Resource mapping between Spinnaker and Kubernetes constructs, as well as the
+introduction of new types of resources, is a lot more flexible in the
+Kubernetes provider V2 for other providers, because of how many types of
+resources Kubernetes supports. Also the Kubernetes extension
+mechanisms&mdash;called [Custom Resource Definitions
 (CRDs)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/)&mdash;make
 it easy to build new types of resources, and Spinnaker accomodates that by
 making it simple to extend Spinnaker to support a user's CRDs.
 
-## Workloads ≈ Server Groups
+## Terminology Mapping
 
-## Services, Ingresses ≈ Load Balancers
+It is worth noting that the resource mapping exists primarily to render
+resources in the UI according to Spinnaker conventions. It does not affect how
+resources are deployed or managed.
 
-## NetworkPolicies ≈ Security Groups
+There are three major groupings of resources in Spinnaker, Server Groups, Load
+Balancers, and Security Groups. They correspond to Kubernetes resource kinds as
+follows:
+
+* Workloads ≈ Spinnaker Server Groups
+* Services, Ingresses ≈ Load Balancers
+* NetworkPolicies ≈ Security Groups
+
+## Resource Management Policies
+
+How you manage the deployment and updates of a Kubernetes resource is dictated
+by its kind, via the policies that apply to a particular kind. Below are
+descriptions of these policies, followed by a mapping of kinds to policies.
+
+* __Operations__
+
+  There are several operations that can be implemented by each kind:
+
+  * _Deploy:_ Can this resource be deployed and redeployed? It's worth
+    mentioning that all deployments are carried out using `kubectl apply` to
+    capitalize on `kubectl`'s three-way merge on deploy. This is done to
+    accomodate running
+  * _Delete:_ Can this resource be deleted?
+  * _Scale:_  (Workloads only) Can this resource be scaled to a desired replica
+    count?
+  * _Undo Rollout:_ (Workloads only) Can this resource be rolled back/forward
+    to an existing revision?
+  * _Pause Rollout:_ (Workloads only) When rolling out, can the rollout be
+    stopped?
+  * _Resume Rollout:_ (Workloads only) When a rollout is paused, can it be
+    started again?
+
+* __Versioning__
+
+  If a resource is "versioned", it will always be deployed with a new sequence
+  number `vNNN`. This is important for resources like ConfigMaps and
+  ReplicaSets, which don't have their own built-in update policy like
+  Deployments or StatefulSets do. Making an edit to the in-place, rather than
+  redeploying can have unexpected results, and delete prior history.
+  Regardless, whatever the policy is, it can be overriden during a deploy
+  manifest stage.
+
+* __Stability__
+
+  This describes under what conditions this kind is considered stable after a
+  new `spec` has been submitted.
+
+## Workloads
```

Comment:
As we discussed, demote these three h2 headings to h3, and add a wrapper heading to link to from above.